scripts/stablex/place_margin_orders.js

```diff
@@ -0,0 +1,91 @@
+const BatchExchange = artifacts.require("BatchExchange")
+const { sendTxAndGetReturnValue } = require("../../test/utilities.js")
+const BN = require("bn.js")
+
+const argv = require("yargs")
+  .option("tokens", {
+    describe: "Collection of trusted tokens",
+  })
+  .option("accountId", {
+    describe: "Account index of the order placer",
+  })
+  .option("margin", {
+    type: "float",
+    describe: "Percentage increase required for trade (fees not accounted)",
+    default: 0.2,
+  })
+  .option("sellAmount", {
+    type: "float",
+    describe: "Maximum sell amount (considered infinite if empty)",
+    default: 100,
+  })
+  .option("expiry", {
+    type: "int",
+    describe: "Maximum auction batch for which these orders are valid",
+    default: 2**32 - 1
+  })
+  .demand(["tokens", "accountId"])
+  .help(false)
```

Comment:
Help me!

Reply:
I''l think about it!

Reply:
I added a little bit of help here... Could probably be better.

Reply:
I meant 
```suggestion
  .help(true)
```

Reply:
Oh. ok and what does that do?